perllib/FixMyStreet/App/Controller/Around.pm

```diff
@@ -195,6 +195,7 @@ sub display_location : Private {
                 colour    => $colour,
                 id        => $p->id,
                 title     => $p->title_safe,
+                problem   => $p,
```

Comment:
If we're passing this in, should the other attributes just hang off it? Probably not worth it.